models/order.js

```diff
@@ -1,6 +1,7 @@
 import mongoose from 'mongoose';
 import { Schema } from 'mongoose';
 import constants  from '../src/constants';
+import Item from './item';
 
 const Entry = new Schema({
   item_id: { type: Schema.ObjectId, ref: 'Item' },
```

Comment:
Are we using references here but nested schema for `Order.entries` because there shouldn't be multiple instances of any given unique `Item`?